luigi_pipeline/seqr_loading.py

```diff
@@ -18,14 +20,61 @@ class SeqrVCFToMTTask(HailMatrixTableTask):
     reference_mt_path = luigi.Parameter(description='Path to the matrix table storing the reference variants.')
     sample_type = luigi.ChoiceParameter(choices=['WGS', 'WES'], description='Sample type, WGS or WES', var_type=str)
     validate = luigi.BoolParameter(default=True, description='Perform validation on the dataset.')
+    dataset_type = luigi.ChoiceParameter(choices=['VARIANTS', 'SV'], default='VARIANTS', description='VARIANTS or SV.')
 
     def run(self):
         mt = self.import_vcf()
         if self.validate:
             self.validate_mt(mt, self.genome_version, self.sample_type)
-        # TODO: Modify MT.
+
+        mt = hl.split_multi(mt)
+        mt = HailMatrixTableTask.run_vep(mt, self.genome_version, self.vep_runner)
+        mt = self.derive_fields(mt, self.dataset_type)
+        mt.describe()
         mt.write(self.output().path)
 
+    @staticmethod
+    def derive_fields(mt, dataset_type):
+        """
+        Transforms a mt with fields derived from `vep`, `locus`, `alleles`, `position`, etc.
+        :param mt: matrix table with required fields (e.g. `vep`, `locus`, etc.)
+        :param dataset_type: either `VARIANTS` or `SV`
+        :return: mt with selected derived fields
+        """
+        mt = mt.annotate_rows(
+            sortedTranscriptConsequences=vep.get_expr_for_vep_sorted_transcript_consequences_array(mt.vep))
+        mt = mt.annotate_rows(
+            docId=variant_id.get_expr_for_variant_id(mt, 512),
+            variantId=variant_id.get_expr_for_variant_id(mt),
+            contig=variant_id.get_expr_for_contig(mt.locus),
+            pos=variant_id.get_expr_for_start_pos(mt),
+            start=variant_id.get_expr_for_start_pos(mt),
+            end=variant_id.get_expr_for_end_pos(mt),
+            ref=variant_id.get_expr_for_ref_allele(mt),
+            alt=variant_id.get_expr_for_alt_allele(mt),
+            xpos=variant_id.get_expr_for_xpos(mt.locus),
+            xstart=variant_id.get_expr_for_start_pos(mt),
```

Comment:
Good catch, I didn't notice that xpos and pos were different. @mike-w-wilson can you look through this and the original to see if I missed anything else?

Reply:
`get_expr_for_filtering_allele_frequency` is used in v01/load_dataset_to_es.py. Not sure if you were waiting to define that elsewhere. Otherwise it looks good to me.

Reply:
Yes, good catch. I remembered I left it out because I was confused since I remembered FAF was something in the gnomad fields now. Let's remember this and revisit it when we do the ref data.